### ISSUE TITLE
Command "cmd on-quit" doesn't work

### ISSUE THREAD

**Issue by hikkidev** [NikitaIvanovV/ctpv#7]
Bug: https://mega.nz/file/BNMkULpZ#CkmpOk7YKSsn6PU84T5RisuUouUHp53b2Uzat83cPyw

You should remove the `cmd` prefix for the new version of lf (r27), otherwise the command will not work

```
set previewer ctpv
set cleaner ctpvclear
&ctpv -s $id
on-quit $ctpv -e $id
```

You should also update the documentation in lf -> https://github.com/gokcehan/lf/wiki/Previews#with-stpv-and-ctpv

System
kitty: version 0.25.2 created by Kovid Goyal
lf: version r27
OS: Arch Linux x86_64 5.18.10-zen1-1-zen
WM: sway v1.7

**Comment by NikitaIvanovV** [comment]
I've updated lf on my machine and everything works correctly. I also didn't dind a commit where they made it so that you don't need `cmd` prefix to define the `on-quit` command. Do you use Kitty terminal? Please tell your kitty terminal and lf versions.

```
kitty -v
lf -version
```

**Comment by hikkidev** [comment]
> Please send your kitty terminal and lf versions.
I've already attached this information above, in the system section.

**Comment by hikkidev** [comment]
Well, just watch this: https://mega.nz/file/9NUz1LbT#TyJF6xHQjkn8n3iSrUtQE8Ar8rkfgIc_LCd_goksw9g

**Comment by NikitaIvanovV** [comment]
> I've already attached this information above, in the system section.

Yeah, I'm sorry, to some reason I didn't notice that.

It was me who requested the addition of `on-quit` command to fix the problem you're experiencing. It was added but it's not a part of the latest release yet.

Please try installing `lf-git` AUR package and see if the problem is fixed.

**Comment by hikkidev** [comment]
No, the same results
![image](https://user-images.githubusercontent.com/52655010/179385120-96ea9a98-551d-4fb4-83f9-ab36adf3aacf.png)

**Comment by hikkidev** [comment]
Configs
```
❯ cat .config/lf/lfrc

set shell sh
set shellopts '-eu'
set ifs "\n"

set scrolloff 10

set icons

# Terminal previewer
set preview true
set previewer ctpv
set cleaner ctpvclear
&ctpv -s $id
on-quit $ctpv -e $id
```
```
❯ cat .config/kitty/kitty.conf
font_family      MesloLGS NF
bold_font        auto
italic_font      auto
bold_italic_font auto

font_size 14.0

adjust_line_height  0
adjust_column_width 0

background_opacity 0.85
```

**Comment by hikkidev** [comment]
I think this is a duplicate of this issue https://github.com/gokcehan/lf/issues/847

**Comment by hikkidev** [comment]
I guess `cmd on-quit` functionality is not yet included in `lf` so I just removed that prefix and the ctpvclear call always happens ¯\_(ツ)_/¯

**Comment by NikitaIvanovV** [comment]
It turns out that there is a minor bug in `lf` where it runs a previewer on quit if `cmd on-quit` is set. I've pushed a fix for it, it's not ideal but should work before the lf bug gets fixed. Could you please update `ctpv-git` and test it?

**Comment by hikkidev** [comment]
Huh
![image](https://user-images.githubusercontent.com/52655010/179684315-1cbb1389-9045-4946-88a0-b44f649b0185.png)

**Comment by hikkidev** [comment]
> I've pushed some changes that should work around the issue, it's not ideal but should work for the time being. 

Yeah, it's strange :)
Record: https://mega.nz/file/pUsDnKha#R-HwPwOXOqt7uvRwwcFMEaRyXKZzS24DNhP2u5ysJd0

**Comment by NikitaIvanovV** [comment]
> Yeah, it's strange :)
Record: https://mega.nz/file/pUsDnKha#R-HwPwOXOqt7uvRwwcFMEaRyXKZzS24DNhP2u5ysJd0

Yeah, the image appears for a second after quitting lf, this is what I called not an idean solution. If you do not use Ueberzug or Chafa to view images (only Kitty terminal), I think you can comment out `cmd on-quit` line in your `lfrc` if the flickering image bothers you.

**Comment by NikitaIvanovV** [comment]
> Huh ![image](https://user-images.githubusercontent.com/52655010/179684315-1cbb1389-9045-4946-88a0-b44f649b0185.png)

What happens if you run `gpg --recv-keys 6E656AC5B97B5133`?

**Comment by hikkidev** [comment]
Oh, never mind. This problem is caused by an empty `/etc/resolv.conf` (systemd-resolved). 
Solved by adding `nameserver 1.1.1.1 # Cloudflare`.

![image](https://user-images.githubusercontent.com/52655010/179895325-30f31163-1628-449a-8bc0-46995aa97c11.png)
